src/components/TheChatInput.vue

```diff
@@ -40,10 +40,16 @@
         },
         computed: {
             ...mapGetters(["account", "localUser"]),
+            canSend() {
```

Comment:
If the message are just spaces and it gets trimmed then it becomes an empty string and not undefined right? So this will always be true i suppose?

Reply:
Empty string  is falsey

Reply:
![image](https://user-images.githubusercontent.com/6184820/92723792-df330580-f369-11ea-9dfd-d99b649034da.png)